### PR TITLE
OSSM-4877 [DOC] Release Notes Update 2.3.7 and 2.2.10

### PR DESCRIPTION
**Pull request by gwynnemonahan**
OSSM [DOC] Release Notes Update 2.3.7 and 2.2.10

Version(s):
4.11+

Issue:
https://issues.redhat.com/browse/OSSM-4877

Link to docs preview:

2.3.7: https://65391--docspreview.netlify.app/openshift-enterprise/latest/service_mesh/v2x/servicemesh-release-notes#new-features-red-hat-openshift-service-mesh-version-2-3-7

2.2.10: https://65391--docspreview.netlify.app/openshift-enterprise/latest/service_mesh/v2x/servicemesh-release-notes#new-features-red-hat-openshift-service-mesh-version-2-2-10

QE review:
- [x] QE has approved this change.

Additional information:

Adds updated component information for 2.3.7 and 2.2.10.

### CODE REVIEW

**Review by pbajjuri20 — APPROVED** [review]

**Comment by pbajjuri20** [comment]
Compared with the Release Readiness doc, everything looks good.

**Review by cbippley — COMMENTED** [review]
LGTM! Please wait for @opayne1 to review before requesting a merge review as I'm still in training. Thanks!

**Review by opayne1 — COMMENTED** [review]
/lgtm

**Review by jldohmann — APPROVED** [review]
lgtm!

**Pull request merged**
